vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl

```diff
@@ -11779,9 +11779,31 @@ quantitykind:PhotonRadiance
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
   rdfs:label "Photon Radiance"@en ;
 .
+quantitykind:PhotosyntheticPhotonFluxDensity
+  rdf:type qudt:QuantityKind ;
+  qudt:hasDimensionVector qkdv:A1E0L-2I0M0H0T-1D0 ;
+  qudt:plainTextDescription "Photosynthetically Active Radiation are the wavelengths of light within the visible range of 400 to 700 nanometers (nm) that are critical for photosynthesis. PPFD measures the amount of PAR light (photons) that arrive at the plant’s surface each second. The PPFD is measured at various distances with a Full-spectrum Quantum Sensor, also known as a PAR meter. Natural sunlight has a PAR value of 900-1500μMol/m2/s when the sun is directly overhead. For a grow light to be effective, it should have PAR values of 500-1500 μMol/m2/s." ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
+  rdfs:label "Photosynthetic Photon Flux Density" ;
+  prov:wasDerivedFrom <https://www.dormgrow.com/par> ;
+  qudt:informativeReference "https://www.gigahertz-optik.com/en-us/service-and-support/knowledge-base/measurement-of-par/"^^xsd:anyURI ;
+  skos:altLabel "PPFD" ;
+  qudt:applicableUnit unit:MicroMOL-PER-M2-SEC ;
+.
+quantitykind:PhotosyntheticPhotonFlux
+  rdf:type qudt:QuantityKind ;
+  qudt:hasDimensionVector qkdv:A1E0L-0I0M0H0T-1D0 ;
+  qudt:plainTextDescription "Photosynthetic Photon Flux (PPF) is a measurement of the total number of photons emitted by a light source each second within the PAR wavelength range and is measured in μmol/s. Lighting manufacturers may specify their grow light products in terms of PPF. It can be considered as analogous to measuring the luminous flux (lumens) of visible light which would typically require the use of an integrating sphere or a goniometer system with spectroradiometer sensor." ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
+  rdfs:label "Photosynthetic Photon Flux" ;
+  prov:wasDerivedFrom <https://www.gigahertz-optik.com/en-us/service-and-support/knowledge-base/measurement-of-par/#WasistPAR> ;
+  qudt:informativeReference "https://www.dormgrow.com/par/"^^xsd:anyURI ;
+  skos:altLabel "PPF" ;
+  qudt:applicableUnit unit:MicroMOL-PER-SEC ;
+.
 quantitykind:PlanckFunction
   a qudt:QuantityKind ;
-  dcterms:description """The \\(\\textit{Planck function}\\) is used to compute the radiance emitted from objects that radiate like a perfect \"Black Body\". The inverse of the \\(\\textit{Planck Function}\\) is used to find the \\(\\textit{Brightness Temperature}\\) of an object. 
+  dcterms:description """The \\(\\textit{Planck function}\\) is used to compute the radiance emitted from objects that radiate like a perfect \"Black Body\". The inverse of the \\(\\textit{Planck Function}\\) is used to find the \\(\\textit{Brightness Temperature}\\) of an object.
```

Comment:
Please add ^^qudt:LatexString ;
at the end of the string.

Reply:
^^qudt:LatexString exists at the end of the description, but the text was somehow spitted into multiple lines. i have now formatted it.